components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/device/constants/Constants.java

```diff
@@ -58,7 +58,6 @@
     public static final String EXPIRY_TIME_PATH = "OAuth.SupportedGrantTypes.SupportedGrantType.ExpiryTime";
     public static final String CONF_KEY_SET = "OAuth.SupportedGrantTypes.SupportedGrantType.KeySet";
     public static final String CONF_USER_CODE_LENGTH = "OAuth.SupportedGrantTypes.SupportedGrantType.UserCodeLength";
-    public static final String RESPNSE_TO_STRING_PATH = "OAuth.SupportedGrantTypes.SupportedGrantType.ResponseToString";
```

Comment:
Any reason to remove this? It's risky to remove public constants.

Reply:
This newly added variable is not required